docs/reference/supervisor.md

```diff
@@ -1400,6 +1399,102 @@ If the specified node is physics-enabled, i.e. it contains a [Physics](physics.m
 This function could be useful for resetting the physics of a solid after changing its translation or rotation.
 To stop the inertia of all available solids please refer to [this section](#wb_supervisor_simulation_reset_physics).
 
+
+---
+
+#### `wb_supervisor_node_set_joint_position`
+
+%tab-component "language"
+
+%tab "C"
+
+```c
+#include <webots/supervisor.h>
+
+void wb_supervisor_node_set_joint_position(WbNodeRef node, double position, int index);
+```
+
+%tab-end
+
+%tab "C++"
+
+```cpp
+#include <webots/Node.hpp>
+
+namespace webots {
+  class Node {
+    void setJointPosition(double position, int index = 1);
```

Comment:
```suggestion
    void setJointPosition(double position, int index = 0);
```

Reply:
On the C++ controller API the default value being applied is 0, not 1. Also, does the C version not impose the same? (and therefore python etc)

Reply:
In joints the "position" fields are called (1), 2, and 3.
So the index here is correct and I just changed the code that was not updated.

C, Java, Matlab and ROS APIs don't support default values.
But I will add in the default value in the Python API.